src/any_circuit_element.ts

```diff
@@ -56,6 +56,7 @@ export const any_circuit_element = z.union([
   pcb.pcb_port_not_matched_error,
   pcb.pcb_fabrication_note_path,
   pcb.pcb_fabrication_note_text,
+  pcb.autorouting_error,
```

Comment:
```suggestion
  pcb.pcb_autorouting_error,
```

Are we not following this convention for this autorouting_error?

cc: @seveibar

Reply:
Oh good point!!!!

Reply:
Cc @Abse2001

Reply:
Should we follow it I wasn't sure if autorouter should be called pcb error?